crates/context/src/config.rs

```diff
@@ -1,15 +1,7 @@
-use camino::Utf8PathBuf;
 use serde::{Deserialize, Serialize};
+use url::Url;
 
-#[derive(Clone, Debug, Deserialize, Serialize)]
-#[non_exhaustive]
-pub struct ApplicationConfig {
-    pub dir: Utf8PathBuf,
-}
-
-impl ApplicationConfig {
-    #[must_use]
-    pub const fn new(dir: Utf8PathBuf) -> Self {
-        Self { dir }
-    }
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct ContextConfig {
+    pub relayer: Url,
 }
```

Comment:
No docs...

Reply:
Can you provide an example of docs for this? Although I agree we need to document stuff, I am not sure what do you expect in this case

Reply:
There's no time for this, and these pieces are under active development (iterated on in successive PRs), documenting every iteration in code will only slow us down further, and I'm out next week.